pyproject.toml

```diff
@@ -1,7 +1,51 @@
 [build-system]
-requires = ["setuptools", "wheel", "oldest-supported-numpy", "packaging>20.9"]
+requires = ["setuptools>=61.0", "wheel", "oldest-supported-numpy", "packaging>20.9"]
 build-backend = "setuptools.build_meta"
 
+[project]
+name = "shap"
+description = "A unified approach to explain the output of any machine learning model."
+readme = "README.md"
+license = {text = "MIT License"}
+authors = [
+    {name = "Scott Lundberg", email = "slund1@cs.washington.edu"},
+]
+requires-python = ">=3.7"
+dependencies = [
+  'numpy', 'scipy', 'scikit-learn', 'pandas', 'tqdm>=4.27.0', 'packaging>20.9',
```

Comment:
at the risk of bikeshedding, should there be line breaks or some sort of consistent toml formatting here?